client/boot/project/wordpress-com.js

```diff
@@ -32,19 +32,21 @@ const config = require( 'config' ),
 import { getSelectedSiteId, getSectionName } from 'state/ui/selectors';
 import { setNextLayoutFocus, activateNextLayoutFocus } from 'state/ui/layout-focus/actions';
 
-function renderLayout( reduxStore ) {
-	const Layout = require( 'controller' ).ReduxWrappedLayout;
+function renderLayoutPromise( reduxStore ) {
```

Comment:
Yeah, I see what you mean. Changed it back to `renderLayout` for now. I think I was trying to make it clear that it was async, but I don't think that's necessary.